### PR TITLE
Longw/3.1.16 update releasenotes

### PR DESCRIPTION
**Pull request by wanlonghenry**
Update 3.1.16 release notes for mooncake mcr change

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ additional questions or comments.
 
 ## Release History
 
-### 11/29/2023 -
-##### Version microsoft/oms:3.1.15 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
-##### Version microsoft/oms:win-3.1.15 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.15 (windows)
+### 12/08/2023 -
+##### Version microsoft/oms:3.1.16 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
+##### Version microsoft/oms:win-3.1.16 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.16 (windows)
```
Comment (pfrcks) [inline ReleaseNotes.md]: win image is still at 3.1.15

File: ReleaseNotes.md [hunk 2]
```diff
@@ -32,8 +32,10 @@ additional questions or comments.
   - Golang - 1.20.5
 ##### Code change log
 ## What's Changed
-## What's Changed
-* update mdsd to 1.28.11 by @pfrcks in https://github.com/microsoft/Docker-Provider/pull/1106
+- AKS Linux
+    * Update mdsd to 1.28.11 by @pfrcks in https://github.com/microsoft/Docker-Provider/pull/1106
```
Comment (pfrcks) [inline ReleaseNotes.md:36]: this change is for both AKS and Arc

File: ReleaseNotes.md [hunk 3]
```diff
@@ -32,8 +32,10 @@ additional questions or comments.
   - Golang - 1.20.5
 ##### Code change log
 ## What's Changed
-## What's Changed
-* update mdsd to 1.28.11 by @pfrcks in https://github.com/microsoft/Docker-Provider/pull/1106
+- Linux
+    * Update mdsd to 1.28.11 by @pfrcks in https://github.com/microsoft/Docker-Provider/pull/1106
+- ARC
```
Comment (ganga1980) [inline ReleaseNotes.md]: nit: rename ARC to Azure Arc K8s Extension

File: ReleaseNotes.md [hunk 4]
```diff
@@ -9,8 +9,8 @@ additional questions or comments.
 
 ## Release History
 
-### 11/29/2023 -
-##### Version microsoft/oms:3.1.15 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
+### 12/08/2023 -
+##### Version microsoft/oms:3.1.16 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
```
Comment (ganga1980) [inline ReleaseNotes.md]: we dont use microsoft/oms:3.1.16 and microsoft/oms:3.1.15 so these can be removed.

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**